frontend/pages/admin/login.tsx

```diff
@@ -0,0 +1,165 @@
+import Button from '@/components/atoms/Button'
+import { CustomIcons } from '@/components/atoms/Icons'
+import InputField from '@/components/atoms/InputField'
+import type { NextPage } from 'next'
+import Head from 'next/head'
+import Image from 'next/image'
+import { useState } from 'react'
+import { Controller, useForm } from 'react-hook-form'
+const { LoadingSpinner } = CustomIcons
+
+type FormValues = {
+    email: string
+    password: string
+}
+
+const AdminLogin: NextPage = () => {
+    // const router = useRouter()
```

Comment:
Remove comment

Reply:
I will use it for my next task which is implementation 🙇